test/integration/options.spec.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var path = require('path');
+var loadOptions = require('../../lib/cli/options').loadOptions;
+
+describe('options', function () {
+  var workingDirectory;
```

Comment:
[Question] Is there a reason not to make this set only once? Does it have to be set before each test?

```suggestion
  const workingDirectory = process.cwd();
```

Reply:
Whenever I change directories for performing tests, I use to always restore the previous working directory in order to not have it interfer with other tests and `beforeEach` functions.

I could entirely drop it or replace it with `beforeAll`/`afterAll`